src/control/server/storage/bdev/backend.go

```diff
@@ -102,18 +102,14 @@ func (w *spdkWrapper) init(log logging.Logger, initShmID ...int) (err error) {
 	}
 	defer restore()
 
-	shmID := 0
-	if len(initShmID) > 0 {
-		shmID = initShmID[0]
-	}
+	log.Debugf("spdk init opts: %v", spdkOpts)
 
 	// provide empty whitelist on init so all devices are discovered
```

Comment:
This comment may be out of date now, or at least misplaced.

Reply:
the below comment applies to this, include/exclude filtering has already been applied in prepare